packages/website/docs/manual/getting-started.md

```
@@ -32,6 +32,182 @@ Copyright (c) JGraph Ltd 2006-2020
+    target: vertex02,
+    value: 'edge',
+    style: {
+      edgeStyle: 'orthogonalEdgeStyle',
+      rounded: true,
+    },
+  });
+});
+```
+If you paste this code into your application, you should have minimal graph displayed with a rectangle connected to an orange circle.
+
+![](./assets/getting-started/first-graph.gif)
```

Markdownlint (0.35.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)